org.eclipse.xtext.xbase/emf-gen/org/eclipse/xtext/xbase/XbasePackage.java

```diff
@@ -1901,6 +1919,17 @@
 	 */
 	EReference getXIfExpression_Else();
 
+	/**
+	 * Returns the meta object for the attribute '{@link org.eclipse.xtext.xbase.XIfExpression#isConditionalExpression <em>Conditional Expression</em>}'.
+	 * <!-- begin-user-doc -->
+	 * <!-- end-user-doc -->
+	 * @return the meta object for the attribute '<em>Conditional Expression</em>'.
+	 * @see org.eclipse.xtext.xbase.XIfExpression#isConditionalExpression()
```

Comment:
what is isConditionalExpression good for?

Reply:
is that ?: ?

Reply:
It allows to detect if we had a ternary expression or an if expression. This will sport warnings and quickfixes.